docs/DeveloperGuide.md

```diff
@@ -93,7 +93,14 @@ For example, the `Money` component is defined as an abstract class that is exten
 
 ### UI Component
 
-<!-- {For Huilin as she is most familiar} -->
+The `UI` component consists following classes: [`UI.java`
+](https://github.com/AY2122S2-CS2113T-T10-2/tp/blob/master/src/main/java/seedu/planitarium/global/UI.java)
+
+The UI component standarizes the output. It provides customized output for `PlanITarium` with `static` method, including
+printing welcome message and exit message as well as chatbox pattern. UI is created when initializing and is called by 
+main during execution. 
```

Comment:
Phrasing should be more standardized / in-line with other component's descriptions